packages/mini-browser/src/node/mini-browser-endpoint.ts

```diff
@@ -190,7 +190,7 @@ export class MiniBrowserEndpoint implements BackendApplicationContribution, Mini
 }
 
 // See `EditorManager#canHandle`.
-const CODE_EDITOR_PRIORITY = 100;
+const CODE_EDITOR_PRIORITY = 200;
```

Comment:
If you change this, we would open, for instance, an SVG as an image, be we want to open the source.

Reply:
I will make a new PR for this issue.